install.php

```diff
@@ -131,7 +131,7 @@
 
   $sqls[] = "
   INSERT INTO `".$prefix."users` (`userid`, `username`, `display_name`, `password`, `email`, `key`, `validated`, `groupid`, `lastactive`, `showavt`, `banned`, `regtime`) VALUES
-(1, 'admin', 'Admin', '7110eda4d09e062aa5e4a390b0a572ac0d2c0220', 'admin@gmail.com', '', '1', 4, ".time().", 1, 0, ".time().");";
+(1, 'admin', 'Admin', '$2y$10\$PGWUi93Gf1D/Z851bMJxDOrLYvXj.r2MFWCNATT1TeJ29S8hfHp4i', 'admin@gmail.com', '', '1', 4, ".time().", 1, 0, ".time().");";
```

Comment:
Yes, indeed. That is the hash of default admin password, 1234. Password_verify() function will be able to verify it even if the password_hash() uses a different hash algorithm in the future because that hash contains information about which hash,salt etc were used for its generation.